app/src/main/java/com/example/projectjuicyfruit/adapters/ItemsAdapter.kt

```diff
@@ -20,7 +22,12 @@ class ItemsAdapter(private var items: MutableList<PetDetails>) :
         when (holder) {
             is ItemsViewHolder -> {
                 if (position != RecyclerView.NO_POSITION) {
-                    holder.bind(items[position])
+                    holder.bind(items[position],holder)
```

Comment:
We pass it in because of the navigation library. In order to set an onClick listener on the  detail button, We need access to a view. That's when we call : **view.findNavController().navigate(R.id.next_action, null)**